example/obstacle_detector.py

```diff
@@ -0,0 +1,52 @@
+import cv2
+import core
+import numpy as np
+import math
+import algorithms
+import time
+
+# Define depth img/data so we can use them for OpenCV window callbacks
+depth_img = None
+depth_data = None
+
+
+def click_print_depth(event, x, y, flags, param):
+    global depth_img, depth_data
+    if event == cv2.EVENT_LBUTTONDOWN:
+        print(depth_img[y][x])
+        print(depth_data.get_data()[y][x])
+
+
+def main():
```

Comment:
Add documentation to example code and remove commented out code and empty comment